frontend/src/classes/Campaign.ts

```diff
@@ -40,6 +41,7 @@ export class Campaign {
     this.sentAt = input['sentAt']
     this.statusUpdatedAt = input['statusUpdatedAt']
     this.protect = input['protect']
+    this.hasFailedRecipients = input['has_failed_recipients']
```

Comment:
The backend returns the following payload:
```
{
   "campaigns":[
      {
         ...
         "statistic":{
            "has_failed_recipients":true
         }
      }
   ],
   "total_count":1
}
```

So `input['has_failed_recipients']` will be undefined.